datumaro/plugins/coco_format/extractor.py

```diff
@@ -124,6 +147,42 @@ def _load_items(self, loader):
 
         return items
 
+    def _load_panoptic_items(self, config, panoptic_images):
+        import numpy as np
+        items = OrderedDict()
+
+        def bgr2id(color):
+            return color[:, :, 2] + 256 * color[:, :, 1] + 256 * 256 * color[:, :, 0]
+
+        imgs_info = {}
+        for img in config['images']:
+            imgs_info[img['id']] = img
+
+        for ann in config['annotations']:
+            img_id = int(ann['image_id'])
+            image_path = osp.join(self._images_dir, imgs_info[img_id]['file_name'])
+            image_size = (imgs_info[img_id].get('height'), imgs_info[img_id].get('width'))
+            image = Image(path=image_path, size=image_size)
+            anns = list()
+
+            panoptic_image_path = osp.join(panoptic_images, ann['file_name'])
+            panoptic_format = np.array(load_image(panoptic_image_path, dtype=np.uint32))
```

Comment:
load_image is expected to return a numpy array

Reply:
fixed